apps/backend/prisma/schema.prisma

```diff
@@ -23,6 +23,7 @@ model User {
   createdAt             DateTime    @default(now())
   notificationUserId    String?
   notificationUsername  String?
+  notificationsEnabled  Boolean
```

Comment:
This should be nullable, and make sure you delete the migration which added this as a nonnullable field otherwise we will delete all users lol

Reply:
oh jesus

Reply:
how often do we snapshot the db?

Reply:
Addressed: https://github.com/cursive-team/connections/pull/177/files#diff-8ce01239249acace801a09d07aaffa847a31687655eebfa2f3715a2f182bc4a7R26

Reply:
I'll now handle + backfill the null case.